conda-forge.yml

```diff
@@ -6,4 +6,5 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+test: native_and_emulated
 test_on_native_only: true
```

Comment:
should both test keys be in here at the same time

Reply:
`test_on_native_only` is deprecated in favor of the one added above, but I don't have access to edit. @hmaarrfk do you mind making the change? 🙂